packages/CollapsibleChecklists/stories/examples/Basic.js

```diff
@@ -0,0 +1,138 @@
+            { isChecked: false, isDisabled: false, name: "LA Lakers" },
+            { isChecked: false, isDisabled: true, name: "LA Clippers" },
+            { isChecked: false, isDisabled: false, name: "Sacramento Kings" },
+          ],
+        },
+        {
+          title: "Hockey",
+          teams: [
+            { isChecked: true, isDisabled: false, name: "Anaheim Ducks" },
+            { isChecked: true, isDisabled: false, name: "LA Kings" },
+            { isChecked: true, isDisabled: false, name: "San Jose Sharks" },
+          ],
+        },
+        {
+          title: (
+            <span>
+              Baseball
+              <span
+                onFocus={() => {}}
+                onMouseOver={handleHover}
+                style={{ paddingLeft: "4px", color: "silver", cursor: "pointer" }}
+              >
+                <InfoIcon />
+              </span>
+            </span>
+          ),
+          teams: [
+            { isChecked: false, isDisabled: false, name: "Anaheim Angels" },
+            { isChecked: false, isDisabled: false, name: "LA Dodgers" },
+            { isChecked: false, isDisabled: false, name: "Oakland Athletics" },
+            { isChecked: false, isDisabled: false, name: "San Diego Padres" },
+            { isChecked: true, isDisabled: false, name: "San Francisco Giants" },
+          ],
+        },
+      ],
+    },
+    {
+      state: "Ohio",
+      sports: [
+        {
+          title: <span>Football (expand to retrieve from API)</span>,
+          teams: [
+            // This will later be retrieved from an API on expand
+          ],
+        },
+      ],
+    },
+  ]);
+
+  const handleExpand = () => {
+    setTimeout(() => {
+      const newSportsData = sportsData.slice(0);
+      newSportsData[1].sports[0].teams = [
+        { isChecked: false, isDisabled: false, name: "Cincinnati Bengals" },
+        { isChecked: false, isDisabled: false, name: "Cleveland Browns" },
+      ];
+      setSportsData(newSportsData);
+    }, 2000);
+  };
+
+  function handleOnChange(changedItemsArray) {
+    console.log("handleOnChange");
+    const newSportsData = sportsData.slice(0);
+
+    newSportsData.forEach(newSportsDatum => {
+      newSportsDatum.sports.forEach(sport => {
+        sport.teams.forEach(team => {
+          const thisTeamWasChanged = changedItemsArray.filter(changedItem => changedItem.key === team.name).length > 0;
+
+          if (thisTeamWasChanged) {
+            team.isChecked = !team.isChecked; // eslint-disable-line
+          }
+        });
+      });
+    });
+
+    setSportsData(newSportsData);
+  }
+
+  function renderTeams(teams) {
+    return teams.map(team => (
+      <CollapsibleChecklists.Item key={team.name} isChecked={team.isChecked} isDisabled={team.isDisabled}>
+        {team.name}
+      </CollapsibleChecklists.Item>
+    ));
+  }
+
+  function renderOhioFootballTeams() {
+    if (sportsData[1].sports[0].teams.length === 0) {
```

Comment:
perhaps create variables for the items with namings like `const firstSport = sports[1].sports[0]` so that can avoid setting the array index in multiple places and easy to follow